test/rubygems/test_gem_version.rb

```diff
@@ -41,6 +41,11 @@ def test_class_create
     assert_equal v('1.1'), Gem::Version.create(ver)
   end
 
+  def test_class_correct
```

Comment:
yes, please make this change, otherwise we could end up accidentally reverting your patch